spec.html

```diff
@@ -25328,7 +25327,7 @@ <h1>
     <emu-clause id="sec-static-semantics-hascallintailposition" type="sdo">
       <h1>
         Static Semantics: HasCallInTailPosition (
-          _call_: unknown,
+          _call_: a Parse Node,
```

Comment:
We could narrow this. Looks like just `CallExpression`, `MemberExpression`, and `OptionalChain`.

Reply:
Yup. I'll change `IsInTailPosition`'s header too, otherwise it isn't obvious that its invocation of `HasCallInTailPosition` satisfies the narrowed type.